### ISSUE TITLE
[BUG] Wrong instance IP selected when using the advanced clustering plugin

### ISSUE THREAD

**Issue by matthewtowen01** [nextgenhealthcare/connect#5272]
**Describe the bug**
When using 2 instances with multiple network interfaces and IP addresses assigned via dhcp. The clustering plugin is picking up the wrong ip and therefor can not communicate on the backend port 10000 for the notification service to work. 

You have to go into the admin console and correct the ip address under settings/clustering after the new server has started. The servers have ephemeral disks and also use an ephemeral server.id. When an instance dies and new one joins the cluster it joins the cluster with a new dhcp address and server.id. The new server will not be able to communicate on the notification port 10000 until you manually go the clustering settings and correct the ip under the cluster settings. 

**Expected behavior**
Clustering plugin should pick the correct instance IP or have a configurable environment variable so you can set the server ip in mirth.properties. 

**Actual behavior**
Wrong ip displayed in clustering plugin settings. 

**Screenshots**
![image](https://user-images.githubusercontent.com/108816723/177579460-a3b50555-eda4-4df4-8a84-afdb0f1c16b9.png)

**Environment (please complete the following information):**
 - OS: Amazon Linux
 - Connect Version 4.0.1

**Comment by pacmano1** [comment]
Is that also true if you set the http and https host explicitly in mirth.properties?

![image](https://user-images.githubusercontent.com/44065187/177586368-27985771-7e83-4fc8-9620-32416677f195.png)

**Comment by matthewtowen01** [comment]
> Is that also true if you set the http and https host explicitly in mirth.properties?
> 
> ![image](https://user-images.githubusercontent.com/44065187/177586368-27985771-7e83-4fc8-9620-32416677f195.png)

Yes I am afraid it is. I have tried setting that and it makes no difference.
